src/pages/people-page/PeoplePage.tsx

```diff
@@ -0,0 +1,118 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { useParams } from 'react-router-dom';
+
+export const PeoplePage = () => {
+  const [peopleList, setPeopleList] = useState<Person[]>([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const { peopleSlug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(res => {
+        const peopleRes: Person[] = [];
+
+        res.map(person => {
```

Comment:
_Array.map_ is used when you want the resulting array, your implementation doesn't do it.
There are two approaches you could take to fix it:
1. Use _Array.map_ directly on peopleRes:
```
const peopleRes: Person[] = res.map(person => {
    ...code goes here
    return { ...person, mother, father }
}
```
2. Keep the code as is, but use _Array.forEach_ instead

The first approach is better as you don't need to create a second array (peopleRes) and push to it.